Sources/EngineKit/HashableExtensions.swift

```diff
@@ -268,7 +339,9 @@ extension ManifestValue: Hashable {
 			return lhs == rhs
 		case let (.nonFungibleLocalIdValue(lhs), .nonFungibleLocalIdValue(rhs)):
 			return lhs == rhs
-		default:
+		case let (.addressReservationValue(lhs), .addressReservationValue(rhs)):
+			return lhs == rhs
+		default /* `switch lhsOuter` above ensures we have handled each `case` */:
```

Comment:
For example, you can put a combined case like `(.boolValue,, _), (.i8Value, _)`etc in the main switch

Reply:
that is an explosion of combinations.

Reply:
No, it's the same number, one entry per case. You have already handled `(.boolValue,, .boolValue)` above, so if you put `(.boolValue,, _)` you know that `_` will be something else.

Reply:
That criticism would also apply to `default`, which would completely break the hasher, but I don't think it's something we need to worry about. It would be very visible if that huge case got moved up to the individual cases, more so than for a lonely default.